tests/performance_tests/algorithm_performance_tests_transitions.py

```diff
@@ -0,0 +1,150 @@
+"""
+This algorithm performance test produces plots for basic transition
+scenarios that contain source, reactor and sink type facilities.
+To use this file, in your command line enter:
+python algorithm_performance_tests_transitions.py [transition type]
+There are currently 3 transition types in place:
+constant, growing and sine.
+"""
+import json
+import re
+import subprocess
+import os
+import sqlite3 as lite
+import copy
+import glob
+import sys
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+import d3ploy.tester as tester
+import d3ploy.plotter as plotter
+import collections
+
+# Delete previously generated files
+direc = os.listdir("./")
+hit_list = (
+    glob.glob("*.json")
+    + glob.glob("*.png")
+    + glob.glob("*.csv")
+    + glob.glob("*.txt")
+    + glob.glob("*.sqlite")
+)
+for file in hit_list:
+    os.remove(file)
+
+ENV = dict(os.environ)
+ENV["PYTHONPATH"] = ".:" + ENV.get("PYTHONPATH", "")
```

Comment:
Why do you need to update the pythonpath here?

Reply:
I just tried running the script without that line and it works the same so I removed it. I'm not sure why it is there too